DDEV Guide - Update Contao Installation DDEV Guide (#1461)
contao/docs: docs/manual/guides/local-installation/ddev.en.md

## Create project

{{% notice tip %}}
The [Contao demo website](https://demo.contao.org/) is maintained for the currently supported Contao versions and can be [optionally 
installed](https://github.com/contao/contao-demo). Via the Contao Manager you can simply select this option during the first installation.
{{% /notice %}}

> {{< tabs groupId="Example">}}

Reviewer: ```suggestion
{{< tabs groupId="ddev-contao-install">}}
```

ddev composer create contao/managed-edition:5.3
```

> After installation, the database access data must be entered in the .env.local. At the same time, we also set up 

Reviewer: ```suggestion
After installation, the database credentials must be entered in the .env.local. At the same time, we also set up 
```

Author: OK

> After installation, the database credentials must be entered in the .env.local. At the same time, we also set up 

Reviewer: ```suggestion
After installation, the database credentials must be entered in the `.env.local`. At the same time, we also set up 
```

The `ddev` binary is not available in the container, so first switch to the host console with `exit`.

> {{% notice tip %}}
> 
> The [Contao Demo Website](https://demo.contao.org/) is maintained for the currently supported Contao versions and can be optionally 
> be installed via Composer. Here you can find information about the [Installation of the Contao Demo](https://github.com/contao/contao-demo).
> 
> {{% /notice %}}

Reviewer: ```suggestion
{{% notice tip %}}
The [Contao Demo Website](https://demo.contao.org/) is maintained for the currently supported Contao versions and can be optionally 
be installed via Composer. Here you can find information about the [Installation of the Contao Demo](https://github.com/contao/contao-demo).
{{% /notice %}}
```
